### PR TITLE
chore(ci): generate android apk

### PR DESCRIPTION
**Pull request by stavares843**
### What this PR does 📖

Adds CI to generate a APK on each PR, that way if the compilation on APK breaks, we can have info about it

Captura de ecrã 2022-10-11, às 01 28 49

I also added CI to automatically add a comment when each APK finishes generating, can add a comment with the artifact like this

Captura de ecrã 2022-10-11, às 01 29 50

and if we add more commits the same comment updates as well

I can remove that part if we don't want the comment from the bot

### CODE REVIEW

**Review by jasonwoodland — APPROVED** [review]

File: .github/workflows/android-apk.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    steps:
+      - name: Checkout 🔖
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js 🔨
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Git Clone Iridium Repo 🗳️
```
Comment (WanderingHogan) [inline .github/workflows/android-apk.yml:18]: drop this block

File: .github/workflows/android-apk.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        run: |
+          cd ..
+          git clone https://github.com/Satellite-im/iridium.git
+      - name: Change permissions on scripts and run Linkscript 🔨
+        run: |
+          chmod +x ./linkscript.sh
+          chmod +x ./cypress/scripts/watch-script.sh
+          chmod +x ./cypress/scripts/sync-node-script.sh
+          ./linkscript.sh
```
Comment (WanderingHogan) [inline .github/workflows/android-apk.yml:27]: drop this line

File: .github/workflows/android-apk.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      - name: Install Yarn Dependencies on Core 🔨
+        run: yarn
```
Comment (WanderingHogan) [inline .github/workflows/android-apk.yml:29]: yarn && ./linkscript.sh

File: .github/workflows/android-apk.yml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+      - name: Create env file - Iridium 🔨
```
Comment (WanderingHogan) [inline .github/workflows/android-apk.yml:31]: drop this block

File: .github/workflows/android-apk.yml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+      - name: Create env file - Core 🔨
+        working-directory: ./
+        run: |
+          touch .env
+          echo ENVIRONMENT="dev" >> .env
+          echo NUXT_ENV_IRIDIUM_SYNC_NODES=${{ secrets.NUXT_ENV_IRIDIUM_SYNC_NODES }} >> .env
+          echo NUXT_ENV_IRIDIUM_APIS=${{ secrets.NUXT_ENV_IRIDIUM_APIS }} >> .env
+          echo NUXT_ENV_IRIDIUM_GATEWAYS=${{ secrets.NUXT_ENV_IRIDIUM_GATEWAYS }} >> .env
+          cat .env
+      - name: Install PNPM dependencies on Iridium 🔨
```
Comment (WanderingHogan) [inline .github/workflows/android-apk.yml:52]: drop this block

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**